### PR TITLE
Added 0.5.0 Changelog

### PR DESCRIPTION
**Pull request by professoralex13**
I did this is segments incase I had a crash while working, only 3 commits

### CODE REVIEW

File: public/changelog.json [hunk 1]
```diff
@@ -1,6 +1,181 @@
...
+                "[General] Added location reporting for the live map - @nistei (Nistei)",
+                "[CDU] Init A now showing computed values and simulated computational delays - @lucky38i (Lucky38i)",
+                "[CDU] Fuel Pred now showing computed values and refreshes when page is updating - @lucky38i (Lucky38i)",
+                "[CDU] Add V1/VR/V2 logic and disagree message, fixed V1/VR/V2 minimum range - @paul92ilm (Lussion)",
+                "[ECAM] Add T.O message category to upper ECAM - @paul92ilm (Lussion)",
+                "[ECAM] Added CABIN PRESS page - @nikotina91",
+                "[ECAM] Add ELEC page - @ramon54321",
+                "[ND] Added altitude constraints in flight level based on the transition altitude provided in MCDU Take Off Performace @ilyeshammadi (Ilyes Hammadi)",
+                "[CDU] Added TYPE I and TYPE II message types and handling - @MisterChocker (Leon)",
+                                "[ND] Add altitue constraint symbol and fix constraint circle color @ilyeshammadi (Ilyes Hammadi)",
```
Comment (ZigTag) [inline public/changelog.json:44]: Inconsistent tabbing

File: public/changelog.json [hunk 2]
```diff
@@ -1,6 +1,181 @@
...
+      ],
+            "fixes": [
+            "[PFD] Fixed V1 not disappear after liftoff - @MisterChocker - (Leon)",
+            "[ECAM] Fixed thrust rating mode START in flight misbehavior - @MisterChocker (Leon)",
+            "[CDU] Fixed wrong VLS calculation - @MisterChocker (Leon)",
+            "[PFD] Fixed wrong managed speed calculation on approach - @MisterChocker (Leon)",
+            "[ECAM] Fixed LDG/TO memo showing at incorrect times - @MMontalto (PiCcy)",
+            "[MISC] Fixed DECEL always show up on left button only - @jokey2k (JoKeY | Markus#0001 on discord)",
+            "[ECAM] Engine bleed valve closes at N2 > 50% - @RichardPilbery (Richard Pilbery)",
+            "[ECAM] Fix thrust lever position indicator on upper ECAM - @RichardPilbery (Richard Pilbery | tricky_dicky3571 on discord)"
```
Comment (ZigTag) [inline public/changelog.json:82]: you forgot a comma

**Review by ZigTag — CHANGES_REQUESTED** [review]

File: public/changelog.json [hunk 3]
```diff
@@ -1,6 +1,181 @@
...
+            ],
+            "fixes": [
+                "[PFD] Fixed V1 not disappear after liftoff - @MisterChocker - (Leon)",
+                "[ECAM] Fixed thrust rating mode START in flight misbehavior - @MisterChocker (Leon)",
+                "[CDU] Fixed wrong VLS calculation - @MisterChocker (Leon)",
+                "[PFD] Fixed wrong managed speed calculation on approach - @MisterChocker (Leon)",
+                "[ECAM] Fixed LDG/TO memo showing at incorrect times - @MMontalto (PiCcy)",
+                "[MISC] Fixed DECEL always show up on left button only - @jokey2k (JoKeY | Markus#0001 on discord)",
+                "[ECAM] Engine bleed valve closes at N2 > 50% - @RichardPilbery (Richard Pilbery)",
+                "[ECAM] Fix thrust lever position indicator on upper ECAM - @RichardPilbery (Richard Pilbery | tricky_dicky3571 on discord)"
```
Comment (ZigTag) [inline public/changelog.json]: another missing comma

**Review by ZigTag — CHANGES_REQUESTED** [review]

**Review by ZigTag — APPROVED** [review]
Code is syntactically correct

**Review by Benjozork — APPROVED** [review]
LGTM

**Pull request merged**